lib/subprocess.js

```diff
@@ -120,7 +175,7 @@ class SubProcess extends EventEmitter {
         // comes in chunks and a line could come in two different chunks, so
         // we have logic to handle that case (using this.lastLinePortion to
         // remember a line that started but did not finish in the last chunk)
-        for (const [streamName, streamData] of _.toPairs(streams)) {
+        for (const [streamName, streamData] of /** @type {[['stdout', string], ['stderr', string]]} */(_.toPairs(streams))) {
```

Comment:
I think `Object.entries()` (which would be comparable) also suffers from this.  We _can_ use `Object.entries` if we want, since we're targeting node.js v14